README.md

```diff
@@ -1,23 +1,22 @@
 # linter-pyflakes
 
-This linter plugin for [Linter][] provides an interface to
-[pyflakes](https://launchpad.net/pyflakes). It will be used with python files.
+[![Build Status](https://img.shields.io/travis/AtomLinter/linter-pyflakes.svg)](https://travis-ci.org/AtomLinter/linter-pyflakes)
+[![Plugin installs!](https://img.shields.io/apm/dm/linter-pyflakes.svg)](https://atom.io/packages/linter-pyflakes)
+[![Package version!](https://img.shields.io/apm/v/linter-pyflakes.svg)](https://atom.io/packages/linter-pyflakes)
+[![Dependencies!](https://img.shields.io/david/AtomLinter/linter-pyflakes.svg)](https://david-dm.org/AtomLinter/linter-pyflakes)
 
-## Installation
+This package will lint the opened `.py` files in Atom through [pyflakes](https://launchpad.net/pyflakes).
 
-Linter package must be installed in order to use this plugin. If Linter is not
-installed, please follow the instructions [here][linter].
+The [changelog](https://github.com/AtomLinter/linter-pyflakes/blob/master/CHANGELOG.md) lists the changes for each release.
 
-### pyflakes installation
+## Installation
 
-Before using this plugin, you must ensure that `pyflakes` is installed on your
-system. To install `pyflakes`, do the following:
+Before using this package, you must ensure that `pyflakes` is installed on your system. To install `pyflakes`, do the following:
 
-1.  Install [pyflakes](https://launchpad.net/pyflakes) by typing the following
-    in a terminal:
+1. On Linux:
 
-    ```ShellSession
-    pip install pyflakes
+    ```text
```

Comment:
[`ShellSession`](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3297) is actually the correct choice here, as it's intended to denote a block of text from a shell session.